docs/maintenance/Releases.mdx

```diff
@@ -26,15 +24,15 @@ Per [Users > Releases > Major Releases](../users/Releases.mdx#major-releases), w
    - Under `push:` > `branches:` at the beginning of the file, add an `- v${major}` list item.
    - Add a `publish_canary_version_v${major}` step the same as `publish_canary_version` except:
      - Change the `if` condition's branch check to: `if: github.ref == 'refs/heads/v${major}'`.
-     - Its publish command should be `npx lerna publish premajor --loglevel=verbose --canary --exact --force-publish --yes --dist-tag rc-v${major}`.
+     - Its publish command should be `npx nx publish premajor --loglevel=verbose --canary --exact --force-publish --yes --dist-tag rc-v${major}`.
```

Comment:
I forgot to update this when I updated the current implementation

Reply:
@JamesHenry sorry, I'm not following - is there more you want to do?

Reply:
Yeah this doc needs to be updated to match the command I applied in the ci.yml, sorry I remembered when on mobile so noted this and I'm on mobile again :D

Reply:
😂 

I can update the doc and then merge, thanks!